login/lib/main.js

```diff
@@ -36,8 +36,13 @@ function main() {
             if (!servicePrincipalId || !servicePrincipalKey || !tenantId || !subscriptionId) {
                 throw new Error("Not all values are present in the creds object. Ensure clientId, clientSecret, tenantId and subscriptionId are supplied");
             }
-            throwIfError(utility_1.execSync("az", "login --service-principal -u \"" + servicePrincipalId + "\" -p \"" + servicePrincipalKey + "\" --tenant \"" + tenantId + "\""));
-            throwIfError(utility_1.execSync("az", "account set --subscription \"" + subscriptionId + "\""));
+            let option = {
+                silent: true,
+                outStream: process.stdout,
+                errStream: process.stderr
+            };
+            throwIfError(utility_1.execSync("az", "login --service-principal -u \"" + servicePrincipalId + "\" -p \"" + servicePrincipalKey + "\" --tenant \"" + tenantId + "\"", option));

```

Comment:
What is the experience now for users when the login fails, given that we are passing silent option.

Reply:
this is the output 
![image](https://user-images.githubusercontent.com/8316321/63632637-f36d6d00-c657-11e9-9da2-28950e89cb6d.png)
No creds are shown here as well.